packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/Detail/AdvancedTxDetails/IODetails/TransactionDetailsModal.tsx

```
@@ -0,0 +1,116 @@
+                            </Paragraph>
+                        </>
+                    ))}
+                    {/* Display Solana token transfers if present */}
+                    {network?.networkType === 'solana' &&
+                        tx.tokens.length > 0 &&
+                        tx.tokens.map((token, index) => (
+                            <>
+                                <Translation
+                                    id="TR_TOKEN_TRANSFERS"
+                                    values={{ standard: token.standard }}
+                                />{' '}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)